examples/js/exporters/GLTFExporter.js

```diff
@@ -1095,21 +1101,26 @@ THREE.GLTFExporter.prototype = {
 
 				if ( targets.length > 0 ) primitive.targets = targets;
 
-				var material = processMaterial( materials[ groups[ i ].materialIndex ] );
-
-				if ( material !== null ) {
+				if ( geometry.index !== null ) {
 
-					primitive.material = material;
+					primitive.indices = processAccessor( geometry.index, geometry, groups[ i ].start, groups[ i ].count );
 
 				}
 
-				if ( geometry.index !== null ) {
+				// Skip meshes without exportable attributes
+				if ( Object.keys(primitive.attributes).length > 0 ) {
```

Comment:
You need spaces after and before `primitive.attributes`

Reply:
Thanks! done